src/language-css/printer-postcss.js

```diff
@@ -1,67 +1,71 @@
-const isHTMLTag = utils.isHTMLTag;
-const isWideKeywords = utils.isWideKeywords;
-const isSCSS = utils.isSCSS;
-const isLastNode = utils.isLastNode;
-const isSCSSControlDirectiveNode = utils.isSCSSControlDirectiveNode;
-const isDetachedRulesetDeclarationNode = utils.isDetachedRulesetDeclarationNode;
-const isRelationalOperatorNode = utils.isRelationalOperatorNode;
-const isEqualityOperatorNode = utils.isEqualityOperatorNode;
-const isMultiplicationNode = utils.isMultiplicationNode;
-const isDivisionNode = utils.isDivisionNode;
-const isAdditionNode = utils.isAdditionNode;
-const isSubtractionNode = utils.isSubtractionNode;
-const isMathOperatorNode = utils.isMathOperatorNode;
-const isEachKeywordNode = utils.isEachKeywordNode;
-const isForKeywordNode = utils.isForKeywordNode;
-const isURLFunctionNode = utils.isURLFunctionNode;
-const isIfElseKeywordNode = utils.isIfElseKeywordNode;
-const hasComposesNode = utils.hasComposesNode;
-const hasParensAroundNode = utils.hasParensAroundNode;
-const hasEmptyRawBefore = utils.hasEmptyRawBefore;
-const isKeyValuePairNode = utils.isKeyValuePairNode;
-const isDetachedRulesetCallNode = utils.isDetachedRulesetCallNode;
-const isPostcssSimpleVarNode = utils.isPostcssSimpleVarNode;
-const isSCSSMapItemNode = utils.isSCSSMapItemNode;
-const isInlineValueCommentNode = utils.isInlineValueCommentNode;
-const isHashNode = utils.isHashNode;
-const isLeftCurlyBraceNode = utils.isLeftCurlyBraceNode;
-const isRightCurlyBraceNode = utils.isRightCurlyBraceNode;
-const isWordNode = utils.isWordNode;
-const isColonNode = utils.isColonNode;
-const isMediaAndSupportsKeywords = utils.isMediaAndSupportsKeywords;
+const {
+  printNumber,
+  printString,
+  hasIgnoreComment,
+  hasNewline
+} = require("../common/util");
+const { isNextLineEmpty } = require("../common/util-shared");
+
+const {
+  builders: {
+    concat,
+    join,
+    line,
+    hardline,
+    softline,
+    group,
+    fill,
+    indent,
+    dedent,
+    ifBreak
+  },
+  utils: { removeLines }
+} = require("../doc");
+
+const {
+  getAncestorNode,
+  getPropOfDeclNode,
+  maybeToLowerCase,
+  insideValueFunctionNode,
+  insideICSSRuleNode,
+  insideAtRuleNode,
+  insideURLFunctionInImportAtRuleNode,
+  isKeyframeAtRuleKeywords,
+  isHTMLTag,
+  isWideKeywords,
+  isSCSS,
+  isLastNode,
+  isSCSSControlDirectiveNode,
+  isDetachedRulesetDeclarationNode,
+  isRelationalOperatorNode,
+  isEqualityOperatorNode,
+  isMultiplicationNode,
+  isDivisionNode,
+  isAdditionNode,
+  isSubtractionNode,
+  isMathOperatorNode,
+  isEachKeywordNode,
+  isForKeywordNode,
+  isURLFunctionNode,
+  isIfElseKeywordNode,
+  hasComposesNode,
+  hasParensAroundNode,
+  hasEmptyRawBefore,
+  isKeyValuePairNode,
+  isDetachedRulesetCallNode,
+  isPostcssSimpleVarNode,
+  isSCSSMapItemNode,
+  isInlineValueCommentNode,
+  isHashNode,
+  isLeftCurlyBraceNode,
+  isRightCurlyBraceNode,
+  isWordNode,
+  isColonNode,
+  isMediaAndSupportsKeywords
+} = require("./utils");
```

Comment:
What do you think about just doing `const utils = require("./utils")` here to decrease the number of local variables?

Reply:
Advantage of many locals is ESLint will catch typos. That isn't the case with `until.foo`.

Reply:
Good point, although it won’t catch typos if you make them in the initial destructuring. (We’d need type checking or ESM to get errors in that case)